app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
   </head>
 
   <body>
-    <%= yield %>
+    <h4><%=flash.notice%></h4>
```

Comment:
add spaces inside ruby injection `<% %>`.
`<%= flash.notice %>` looks better imo

Reply:
don't think that `flash.notice` is correct.
What if you'll have an alert toast?
check pls